hello.py

```diff
@@ -1 +1,6 @@
-print("Hello, world!")
+def hello():
```

Comment:
Can you write a docstring and provide a type hint?

```suggestion
def hello() -> None:
```